### PR TITLE
Update to new Java bindings + small tweaks

### PR DESCRIPTION
**Pull request by TheBlueMatt**

### CODE REVIEW

File: docs/build_node.md [hunk 1]
```diff
@@ -212,19 +191,12 @@ final channel_manager = ChannelManager.constructor_new(
...
-// Assert we were able to read successfully.
-assert channel_manager_read_result instanceof 
-    Result_C2Tuple_BlockHashChannelManagerZDecodeErrorZ
-    .Result_C2Tuple_BlockHashChannelManagerZDecodeErrorZ_OK;
-
-final channel_manager = ((Result_C2Tuple_BlockHashChannelManagerZDecodeErrorZ
-    .Result_C2Tuple_BlockHashChannelManagerZDecodeErrorZ_OK) channel_manager_read_result)
-    .res.b;
+ChannelManagerConstructor channel_manager_constructor = new ChannelManagerConstructor(
+  serialized_channel_manager, channel_monitors.toArray(new byte[1][]),
```
Comment (valentinewallace) [inline docs/build_node.md:195]: Could we add a line:
`final channel_monitors = channel_monitors.toArray(new byte[1][])` with a comment above the line describing what's happening in it?

**Comment by TheBlueMatt** [comment]
Rebased and fixed comments.

File: docs/build_node.md [hunk 2]
```diff
@@ -86,12 +87,18 @@ Persist persister = Persist.new_impl(new Persist.PersistInterface() {
   }
 });
 ```
+
+**Implementation notes:** ChannelMonitors are objects which are capable of responding to on-chain
+events for a given channel. Thus, you will have one ChannelMonitor per channel, identified by the
```
Comment (valentinewallace) [inline docs/build_node.md:92]: nit: backticks around `ChannelMonitor`

**Comment by TheBlueMatt** [comment]
Went ahead and squashed.

**Review by valentinewallace — APPROVED** [review]

**Pull request merged**